OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -45,6 +45,15 @@
     private IPageStatusListener pageStatusListener;
     private String hostForThisPage = null;
 
+    // Tells if the page loading has been finished or not.
+    private boolean loadingFinished = true;
+    // Url will be considered as redirected if it will not be the initial page url requested to load.
+    // For e.x. in case server redirects page url to other hosting url or user click on some link
+    // on page, they will be consider as redirecting urls.
+    private boolean redirect = false;
```

Comment:
Doc should be in javadoc style.

`For e.x.` isn't a word or acronym, either use `e.g.` or `For example`.

`For example, in case the server redirects us to another URL or the user clicks a link on the web-page, it will be considered as a redirect.`